### DISCUSSION TITLE
Vue localization and popup

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9435] [CLI - PWA mode]
Hello Friends

I have a problem to add a line break in a Popup component , am using vue I18n localization to add the line break when am inspecting the elements its showing the showing the sentence in 2 lines but not in the popup component. Can someone please help.
This is the message  wanted to add a line break by using **\n**
 "CancelWarning": "Your content has not been properly saved yet. \n Are you sure you want to leave this page ?",

![image](https://user-images.githubusercontent.com/84211129/119734187-a56c4580-be8b-11eb-97b8-b0bf9fc32a52.png)
![image](https://user-images.githubusercontent.com/84211129/119734218-b452f800-be8b-11eb-9859-954a3e0840ef.png)

[CardCancelPopup.txt](https://github.com/quasarframework/quasar/files/6549739/CardCancelPopup.txt)

**Comment by ontwikkelfabriek** [comment]
@akashramtohul4 

Why not split it to separate `divs`, `spans`  or `p`? 

<div>Your content has not been properly saved yet.</div>
<div>Are you sure you want to leave this page ?</div>
